deploy/helm/sumologic/conf/events/events.conf

```diff
@@ -21,6 +21,11 @@
 <source>
   @type prometheus_output_monitor
 </source>
+<source>
```

Comment:
hah this was what I was asking about at standup

Reply:
oh, I guess I misunderstood at that time.